File: src/main/java/kr/mjc/rachel/basicsyntax/Ranges.java
package kr.mjc.rachel.basicsyntax;

public class Ranges {
  public static void main(String[] args) {
    for (int x = 0; x <= 10; x++) {
      System.out.print(x);
    }
    System.out.println();

    for (int x = 0; x < 10; x++) {
      System.out.print(x);
    }
    System.out.println();

    for (int x = 1; x <= 10; x += 2) {
      System.out.print(x);
    }
    System.out.println();

    for (int x = 9; x >= 0; x -= 3) {
      System.out.print(x);
    }
    System.out.println();
  }
}

File: src/main/kotlin/kr/mjc/jacob/basicsyntax/Ranges.kt
/**
 * https://kotlinlang.org/docs/basic-syntax.html#ranges
 */
package kr.mjc.jacob.basicsyntax

fun main() {
  for (x in 0..10) {  // 0 ~ 10
    print(x)
  }
  println()

  for (x in 0..<10) { // 0 ~ 9
    print(x)
  }
  println()

  for (x in 1..10 step 2) { // 1 3 5 7 9
    print(x)
  }
  println()

  for (x in 9 downTo 0 step 3) { // 9 6 3
    print(x)
  }
}